### PR TITLE
add csi-driver-nvmf sanity test pipeline

### PR DESCRIPTION
**Pull request by MeinhardZhou**
Signed-off-by: Meinhard Zhou <zhouenhua@bytedance.com>

### CODE REVIEW

**Review by andyzhangx — APPROVED** [review]
/lgtm

File: config/jobs/kubernetes-csi/csi-driver-nvmf/OWNERS [hunk 1]
```diff
@@ -0,0 +1,6 @@
+# See the OWNERS docs at https://go.k8s.io/owners
+
+reviews:
+- meinhardzhou
+approvers:
+- meinhardzhou
```
Comment (pohly) [inline config/jobs/kubernetes-csi/csi-driver-nvmf/OWNERS:6]: This file is only going to be useful for manually created jobs: for the automatically generated ones the real change is in `config/jobs/kubernetes-csi/gen-jobs.sh` and there review and approval from others will be needed.

I don't know whether the test-infra owners are okay with individual developers being able to define Prow jobs without them being involved :shrug:

Comment (MeinhardZhou) [reply config/jobs/kubernetes-csi/csi-driver-nvmf/OWNERS]: Well, then I will remove these two files first, and add them when there is an exact message.

**Review by pohly — APPROVED** [review]
/lgtm
/approve

**Pull request merged**